data/hai/hai reveal 5 philosophers.txt

```diff
@@ -876,15 +956,15 @@ mission "Hai Reveal [B12] Devil-Hide Battle"
 	npc save accompany
 		government "Navy (Oathkeeper)"
 		personality escort heroic opportunistic
-		ship "Cruiser (Plasma Anti-Missile)" "N.S. Peacemaker"
+		ship "Cruiser (Quantum Keystone)" "N.S. Peacemaker"
 	npc save accompany
 		government "Republic"
 		personality timid escort opportunistic
-		ship "Auxiliary (Transport)" "N.S. Braveheart"
+		ship "Auxiliary (Quantum Keystone)" "N.S. Braveheart"
 	npc accompany
 		government "Free Worlds"
 		personality escort heroic opportunistic
-		ship "Dreadnought (Jump)" "F.S. Ironwood"
+		ship "Dreadnought (Quantum Keystone)" "F.S. Ironwood"
```

Comment:
As above

Reply:
This was because I moved the variants into "hai reveal 5 philosophers.txt," and the purpose of the incorrect ones variants.txt confused me. I will update the variants.txt versions instead.